Examples/create-windows-cloud-image.ps1

```diff
@@ -12,39 +12,70 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-$ErrorActionPreference = "Stop"
-
 git submodule update --init
-Import-Module ..\WinImageBuilder.psm1
+Import-Module ..\WinImageBuilder.psm1 -Verbose
+Import-Module ..\Config.psm1 -Verbose
+# The path were you want to create the config fille
+$ConfigFilePath ="C\image\config.ini"
```

Comment:
create the config file afterwards and leave the variable definitions as is, because the diff will make more sense, if you don t change the structure of the code